transforms/csdl/preprocess_csdl.xsl

```diff
@@ -6,7 +6,7 @@
     <xsl:strip-space elements="*"/> <!-- Remove empty space after deletions. -->
     <xsl:param name="remove-capability-annotations">True</xsl:param>
     <xsl:param name="add-innererror-description">False</xsl:param>
-    <xsl:param name="csdlVersion">v1_0</xsl:param>
+    <xsl:param name="csdlVersion">v1.0</xsl:param>
```

Comment:
May I suggest that we define custom functions so the comparison doesn't need to be done across the document?

https://stackoverflow.com/questions/16164205/create-custom-xslt-function

Reply:
Can you expound this further? How do you foresee using the `csdlVersion` param with custom functions? What are we aiming to resolve?

Reply:
sure. With this param, every block that needs to use it needs to make the string comparison. Should the value change (like you pointed out from underscore to a period), every comparison will need to be updated. Whereas with a function, all the blocks can call the function (IsV1, IsBeta) and the comparison is only done at a single place.

Reply:
How about this approach?

https://github.com/microsoftgraph/msgraph-metadata/blob/1ac31d90adb4b3a99079ff422e222cdc2c69ff5b/transforms/csdl/preprocess_csdl.xsl#L20-L25